### ISSUE TITLE
Add support for Ubuntu 18.04 (Bionic)

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/agent#211]
We should expand our x86 support to include Ubuntu 18.04 as well, as it is the latest LTS release.

**Comment by kkom** [comment]
Thanks @vpetersson !

Got a bit further on my `aarch64` compilation of Ubuntu 18.04, but now ghostunnel is an issue:

```
➜  ~ sudo apt install -y wott-agent
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 wott-agent : Depends: ghostunnel but it is not installable
E: Unable to correct problems, you have held broken packages.
```

I wonder if the problem is with my architecture this time...

**Comment by vpetersson** [comment]
@kkom, Yes, we need to push some new packages to the repository. @a-martynovich The deb files you're building should do the trick, right?

**Comment by a-martynovich** [comment]
@vpetersson I don't think we're building ghostunnel for aarch64. I can add it though.

**Comment by vpetersson** [comment]
Re-opening this until we've submitted the deb files.

**Comment by a-martynovich** [comment]
Fixed by wottsecurity/ghostunnel#4
